src/System Application/Test/Date and Time/src/UnixTimestampTest.Codeunit.al

```diff
@@ -0,0 +1,83 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.DateTime;
+
+using System.DateTime;
+using System.TestLibraries.Utilities;
+
+codeunit 132980 "Unix Timestamp Test"
+{
+    Subtype = Test;
+
+    var
+        LibraryAssert: Codeunit "Library Assert";
+
+    [Test]
+    procedure GivenDateTimeCreatesGivenTimestampSeconds()
```

Comment:
Minor, I would name the tests with test. E.g. DateTimeCreatesUNIXTIMESTAMP
No need to repeat the GIVEN WHEN THEN

Reply:
That is indeed more concise. Fixed.